sys/linux/auto.txt.const

```diff
@@ -2423,31 +2423,31 @@ TCP_METRICS_CMD_DEL = 2
 TCP_METRICS_CMD_GET = 1
 TCSBRK = 21513, mips64le:21509, ppc64le:536900637
 TCSBRKP = 21541, mips64le:21638
-THERMAL_GENL_ATTR_CDEV = 15
-THERMAL_GENL_ATTR_CDEV_CUR_STATE = 17
-THERMAL_GENL_ATTR_CDEV_ID = 16
-THERMAL_GENL_ATTR_CDEV_MAX_STATE = 18
-THERMAL_GENL_ATTR_CDEV_NAME = 19
-THERMAL_GENL_ATTR_CPU_CAPABILITY = 21
-THERMAL_GENL_ATTR_CPU_CAPABILITY_EFFICIENCY = 24
-THERMAL_GENL_ATTR_CPU_CAPABILITY_ID = 22
-THERMAL_GENL_ATTR_CPU_CAPABILITY_PERFORMANCE = 23
-THERMAL_GENL_ATTR_THRESHOLD = 25
-THERMAL_GENL_ATTR_THRESHOLD_DIRECTION = 27
-THERMAL_GENL_ATTR_THRESHOLD_TEMP = 26
+THERMAL_GENL_ATTR_CDEV = 14
```

Comment:
Oh, that's fun. They seem to have a broken release v6.13 with broken attribute values. Now userspace programs can do random unintentional things. E.g. setting fire for real since it's "THERMAL" interface.